source/_components/switch.tplink.markdown

```diff
@@ -30,6 +30,10 @@ To use your TP-Link switch in your installation, add the following to your `conf
 switch:
   - platform: tplink
     host: IP_ADDRESS
+    name: switch
+  - platform: tplink
+    host: IP_ADDRESS
+    name: another switch
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Also, it's already documented how to add additional devices.